plugins/chat/spec/models/category_channel_spec.rb

```diff
@@ -124,6 +124,13 @@
           channel.validate
           expect(channel.errors.full_messages).to include("Slug is invalid")
         end
+
+        it "does not add a validation error when autogenerated slug ends up empty" do
+          SiteSetting.slug_generation_method = "none"
+          channel.update!(name: "は無効です", slug: nil)
```

Comment:
Correct me if I'm wrong, but `#update!` would raise an exception on validation error vs `#update` which would add to the `errors`. If so there's a bit of a disconnect between this setup and the assertion on the line below.

Reply:
Good point, fixing